src/plugin.scss

```diff
@@ -82,40 +88,21 @@
 
 .vjs-errors-content-container {
     overflow: auto;
-    position: absolute;
+    position: relative;
     padding-bottom: 15px;
-    top: 75px;
+    top: -10px;
```

Comment:
Ok Sure. I could try and refactor the the CSS to have positive values, and have the same effect as the screenshots.